composites/basic/Notification.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+
+import Paper from "./Paper";
+import colors from "../../style-guide/colors.json";
+import { Icon } from "../../composites/Plugin/Shared/components/Icon";
+import { times } from "../../style-guide/svg";
+
+const messages = defineMessages( {
+	buttonAriaLabel: {
+		id: "notification.buttonAriaLabel",
+		defaultMessage: "Dismiss this notice",
```

Comment:
I've used this text because it's the standard text used in WordPress: `Dismiss this notice.`
No objections in changing it, however users will hear different text when navigating through the interface, sometimes `Dismiss this notice.` and sometimes `Dismiss this notification.` 🙂 Not sure hte change will add a great value, at the cost of some potential confusion.

Reply:
Let's keep this consistent with WP: `notice`